.gitignore

```diff
@@ -38,6 +38,7 @@ __pycache__
 
 # Ignore lucene included build
 lucene/
```

Comment:
so I thought the `lucene/` entry was in case of having local Lucene development? It might be worth changing to `/lucene/` or something similar that matches top level only? 

I think its a bit surprising that `lucene/` matches any folder in the file structure.